src/map/utils/gardenutils.cpp

```diff
@@ -275,7 +278,14 @@ namespace gardenutils
             strength += dominantAura / 10;
         }
 
-        strength += (int16)((100 - strength) * (PItem->getStrength() / 32.0f));
+        strength += (int16)((100 - strength) * (PItem->getStrength() / 31.0f));
```

Comment:
I have a previous PR that fixes this a different way: https://github.com/LandSandBoat/server/pull/4522
I chose to increase the RNG to 0-32 and keep the divisor as 32, I guess they chose to drop the divisor to 31 but leave the range from 0-31.  Either is probably fine tbh but we can't have both changes together or it'll be wrong.

Reply:
Thanks for weighing in, I was just pulling things over willy-nilly 👍